frappe/patches/v7_1/rename_chinese_language_codes.py

```diff
@@ -3,6 +3,7 @@
 def execute():
 	frappe.rename_doc('Language', 'zh-cn', 'zh', force=True,
 		merge=True if frappe.db.exists('Language', 'zh') else False)
-	frappe.rename_doc('Language', 'zh-tw', 'zh-TW', force=True)
+    if not frappe.db.exists('Language', 'zh-TW') and frappe.db.exists('Language', 'zh-tw'):
```

Comment:
Thanks can you fix the whitespace?

Also best to run a patch before submitting it!